content-service/server/database/beacons.js

```diff
@@ -27,6 +29,41 @@ module.exports = function(knex) {
       });
   }
 
+  function mapDatabaseResponseToApiResponse(beacon) {
+    const location = JSON.parse(beacon.location);
+    const short = shortId.numToShortId(beacon.id);
+    return {
+      id: short,
+      short_url: resolveUrl(SHORT_URL, short),
+      channel: beacon.channel_name,
+      url: beacon.canonical_url,
+      call_to_action: JSON.parse(beacon.call_to_action),
+      extra_metadata: JSON.parse(beacon.extra_metadata),
+      location:  {
+        latitude: location.coordinates[1],
+        longitude: location.coordinates[0],
+      },
+      is_virtual: beacon.is_virtual,
+    };
+  }
+
+
+  function batchGetBeaconInfo(channelName) {
```

Comment:
Any reason for this naming? `getAllBeaconsForChannel` seemed to make more sense to me.